lib/modules/apostrophe-permissions/lib/strategiesApi.js

```diff
@@ -121,6 +126,13 @@ module.exports = function(self, options) {
                 }
               ]
             });
+
+            // Case #5: doc is restricted to exclude certain people
```

Comment:
Should not add this clause at all if the `$nin` clause would be empty.

Reply:
It cannot be empty because `self.userPermissionNames` will always return `[ 'exclude-view-%current-user-id%', 'exclude-view-%current-group-id%' ]`.